app/views/admin_mailer/send_spam_alert.html.erb

```diff
@@ -3,7 +3,7 @@
   <p>The following accounts have a suspicious level of traffic:</p>
   <% @spam.each_pair do |user_id, info| %>
     <% user = User.find(user_id) %>
-    User <%= style_link(user.login, root_url + user_works_path(user)) %> has a score of 
+    User <%= style_link(user.login, root_url + user_works_path(user).gsub(/^\//, '')) %> has a score of 
```

Comment:
You might be able to do `<%= style_link(user.login, user_works_url(user) %>` here -- user_works_url(user) should give the full URL instead of just the /users/user/works path.